doc/sphinx-guides/source/api/dataaccess.rst

```diff
@@ -46,7 +46,7 @@ The fully expanded example above (without environment variables) looks like this
 
 .. code-block:: bash
 
-  curl -O -J -H X-Dataverse-key:xxxxxxxx-xxxx-xxxx-xxxx-xxxxxxxxxxxx https://demo.dataverse.org/api/access/dataset/:persistentId/?persistentId=doi:10.70122/FK2/N2XGBJ
+  curl -L -O -J -H X-Dataverse-key:xxxxxxxx-xxxx-xxxx-xxxx-xxxxxxxxxxxx https://demo.dataverse.org/api/access/dataset/:persistentId/?persistentId=doi:10.70122/FK2/N2XGBJ
```

Comment:
This is the "fully expanded" example but we should also update the non-expanded example above.

Reply:
Good catch, updated in https://github.com/IQSS/dataverse/commit/4861ccffb9b27996a461f99b1d055799bdca5e8e